bindings/matrix-sdk-ffi/src/timeline.rs

```diff
@@ -270,6 +279,8 @@ pub enum TimelineItemContentKind {
     RedactedMessage,
     Sticker { body: String, info: ImageInfo, url: String },
     UnableToDecrypt { msg: EncryptedMessage },
+    RoomMembership { change: MembershipChange },
```

Comment:
Semi-related, is this for all `m.room.member` events? I'm thinking separate `MembershipChange` and `ProfileChange` variants might make sense (the latter would be a separate PR).

Reply:
Sure we can do that. You put this comment in the FFI crate, but I'm guessing this is valid in the main crate too?

Reply:
Oh, yes 😅